pm/recipient.go

```diff
@@ -15,10 +16,13 @@ import (
 )
 
 var errInsufficientSenderReserve = errors.New("insufficient sender reserve")
+var ErrTicketParamsExpired = errors.New("TicketParams expired")
 
 // maxWinProb = 2^256 - 1
 var maxWinProb = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 256), big.NewInt(1))
 
+var paramsExpirationTime = big.NewInt(1)
```

Comment:
What do you think of 4 blocks? 

This would make the lifetime of a ticket fall within the window of the moving average of the last 5 blocks (iirc) geth uses to suggest the gas price.

Reply:
ff653b187cafa1255db5f151f2a8549c38810560

Reply:
> This would make the lifetime of a ticket fall within the window of the moving average of the last 5 blocks (iirc) geth uses to suggest the gas price.

I don't think we should try to optimize this value based on geth's behavior since the # of past blocks that the gas price estimator will look back is configurable so we should steer away from making too many assumptions about the frequency of suggested gas price changes. I think we should lean towards starting with more breathing room with the expiration time. What about 5 blocks i.e. ~1 min (only 1 more than the current value, but approximating 1 min feels easier to justify)?

Reply:
sounds good, fixed in 0c3596c5